streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -310,7 +313,16 @@ public StreamThread(TopologyBuilder builder,
         log.info("{} Creating producer client", logPrefix);
         this.producer = clientSupplier.getProducer(config.getProducerConfigs(threadClientId));
         log.info("{} Creating consumer client", logPrefix);
-        this.consumer = clientSupplier.getConsumer(config.getConsumerConfigs(this, applicationId, threadClientId));
+
+        Map<String, Object> consumerConfigs = config.getConsumerConfigs(this, applicationId, threadClientId);
+
+        if (!builder.latestResetTopicsPattern().pattern().equals("") || !builder.earliestResetTopicsPattern().pattern().equals("")) {
```

Comment:
Why do we need to update the configs? By using seeks we can simply ignore the underlying configed value right? More specifically, as done below we can check if `consumer.committed` returns anything or not, and if no AND the reset policy is specified, we just use seeks to the specified positions.